source/reference/collection-level-filtering.txt

```diff
@@ -58,39 +44,11 @@ Filters have the following syntax:
           "collections": [
              "<collection-name>"
           ]
-          "databaseRegex": {
-             "pattern": "<regex-pattern>",
-             "options": "<options>"
-          },
-          "collectionRegex": {
-             "pattern": "<regex-pattern>",
-             "options": "<options>"
-          }
-       }
-    ],
-    "excludeNamespaces": [
-       {
-          "database": "<database-name>",
-          "collections": [
-             "<collection-name>"
-          ]
-          "databaseRegex": {
-             "pattern": "<regex-pattern>",
-             "options": "<options>"
-          },
-          "collectionRegex": {
-             "pattern": "<regex-pattern>",
-             "options": "<options>"
-          }
        }
     ]
 
 Filters must include either the ``database`` field or the ``databaseRegex`` field.
```

Comment:
I think we'll want to revert this line too, since regex and exclusion are added together.

Reply:
Fixed